integration-tests/src/test/resources/hadoop/batch_hadoop_indexer.json

```diff
@@ -0,0 +1,126 @@
+{
+  "type": "index_hadoop",
+  "spec": {
+    "dataSchema": {
+      "dataSource": "batchHadoop",
+      "parser": {
+        "type": "string",
+        "parseSpec": {
+          "type": "tsv",
+          "timestampSpec": {
+            "column": "timestamp",
+            "format": "yyyyMMddHH"
+          },
+          "dimensionsSpec": {
+            "dimensions": [
+              "dim1",
+              "dim2",
+              "dim3",
+              "dim4",
+              "dim5",
+              "dim6",
+              "dim7",
+              "dim8",
+              "dim9",
+              "dimA",
+              "dimB",
+              "dimC",
+              "dimD"
+            ]
+          },
+          "columns": [
+            "timestamp",
+            "dim1",
+            "dim2",
+            "dim3",
+            "dim4",
+            "dim5",
+            "dim6",
+            "dim7",
+            "dim8",
+            "dim9",
+            "dimA",
+            "dimB",
+            "dimC",
+            "dimD",
+            "m00000000",
+            "m1111_1111_11111",
+            "m2222222_2222_22222",
+            "m333333_33_3333_333333",
+            "m444444_4444_4444_444444",
+            "m555_555555",
+            "m666_666666"
+          ]
+        }
+      },
+      "metricsSpec": [
+        {
+          "type": "longSum",
+          "name": "m00000000",
+          "fieldName": "m00000000"
+        },
+        {
+          "type": "longSum",
+          "name": "m1111_1111_11111",
+          "fieldName": "m1111_1111_11111"
+        },
+        {
+          "type": "longSum",
+          "name": "m2222222_2222_22222",
+          "fieldName": "m2222222_2222_22222"
+        },
+        {
+          "type": "sketchMerge",
```

Comment:
can you use the new api for sketches described at http://druid.io/docs/0.8.3/development/datasketches-aggregators.html ?